src/readii/process/images/crop.py

```
@@ -0,0 +1,425 @@
+        Method to use to crop the image to the mask. Must be one of "bounding_box", "centroid", "cube, or "pyradiomics".
+    resize_dimensions : tuple[int,int,int]
+        Dimensions to resize the image to.
+    
+    Returns
+    -------
+    cropped_image : sitk.Image
+        Cropped image.
+    cropped_mask : sitk.Image
+        Cropped mask.
+    """
+    if resize_dimensions is None and crop_method is not "pyradiomics":  
```

Ruff (0.8.2)

Use `!=` to compare constant literals

Replace `is not` with `!=`

(F632)